lib/test/t_LinearModelAlgorithm_std.cxx

```diff
@@ -70,6 +70,15 @@ int main(int, char *[])
     LinearModelAlgorithm test(X, Y);
     LinearModelResult result(test.getResult());
     fullprint << "trend coefficients = " << result.getCoefficients() << std::endl;
+
+    // Test various attributes
+    const Point cook(result.getCookDistances());
+    const Point cook_reference = {0.0233296,0.0360369,0.00178903,0.0502183,0.0966701,0.00562596};
+    assert_almost_equal(cook, cook_reference, 1e-5, 0.0);
+
+    const Point leverages(result.getLeverages());
+    const Point leverages_reference = {0.0864939,0.0797831,0.0735447,0.0677578,0.0624023,0.0574582};
+    assert_almost_equal(leverages, leverages_reference, 1e-6, 0.0);
   }
```

Comment:
I just noticed this.
I have some troubles with.  IMHO `cook` data should be of size `100` (same as data sample)
How did you pass this?

Reply:
It is very strange
```
(base) haddad_s@becane:~/dev/openturns/build$ ./lib/test/t_LinearModelAlgorithm_std
Fit y ~ 3 - 2 x + 0.05 * sin(x) model using 20 points (sin(x) ~ noise)
trend coefficients = class=Point name=Unnamed dimension=2 values=[3.01168,-2.00025]
Fit y ~ 1 + 0.1 x + 10 x^2 model using 100 points
trend coefficients = class=Point name=Unnamed dimension=3 values=[0.978992,0.110565,9.99924]
libc++abi.dylib: terminating with uncaught exception of type OT::InvalidArgumentException: A and B must have the same dimension 100 vs 6
Abort trap: 6
(base) haddad_s@becane:~/dev/openturns/build$ ctest -R LinearModelAlgorithm_std
Test project /Users/haddad_s/dev/openturns/build
    Start  451: cppcheck_GeneralLinearModelAlgorithm_std
1/6 Test  #451: cppcheck_GeneralLinearModelAlgorithm_std ..............   Passed    0.12 sec
    Start  453: cppcheck_LinearModelAlgorithm_std
2/6 Test  #453: cppcheck_LinearModelAlgorithm_std .....................   Passed    0.06 sec
    Start  457: cppcheck_GeneralLinearModelAlgorithm_std_hmat
3/6 Test  #457: cppcheck_GeneralLinearModelAlgorithm_std_hmat .........   Passed    0.07 sec
    Start 1001: pyinstallcheck_GeneralLinearModelAlgorithm_std
4/6 Test #1001: pyinstallcheck_GeneralLinearModelAlgorithm_std ........   Passed    0.84 sec
    Start 1003: pyinstallcheck_GeneralLinearModelAlgorithm_std_hmat
5/6 Test #1003: pyinstallcheck_GeneralLinearModelAlgorithm_std_hmat ...   Passed    0.57 sec
    Start 1008: pyinstallcheck_LinearModelAlgorithm_std
6/6 Test #1008: pyinstallcheck_LinearModelAlgorithm_std ...............   Passed    0.58 sec

100% tests passed, 0 tests failed out of 6
```
It strange because it fails but pass through the `ctest`

Reply:
the test can still segfault if the diff is ok as ctest sees the return code of diff

Reply:
Nothing except the error message
OK Understand why this generates troubles
I will drop this part and add try/catch

Reply:
Wow. Epic fail on my part! Have you figured out why the CI passes anyway?

Reply:
Don't know. I added try/catch but no changes.
By the way I drop this block

Reply:
Are you replacing it with something else? Otherwise, I think it would be better to fix it by taking only the first 6 elements of `leverages_reference`.